README.md

```diff
@@ -1612,12 +1616,14 @@ that depends greatly on your skills, the marketplace of candidates, what other o
 have, what the company can pay, what other candidates the company has found, and the
 company’s needs.
 However, this section covers some basics of what to expect with offers, and tips on
-negotiating an offer.
+[negotiating an offer](https://hbr.org/2014/04/15-rules-for-negotiating-a-job-offer).
 
 ### Offers
 
-🔹 Most companies, especially well-established ones, give roughly equal treatment to
-candidates. But even so, sophisticated negotiators can often get better offers.
+🔹 Most companies, especially well-established ones, give roughly equal treatment
+[roughly equal treatment](https://rework.withgoogle.com/guides/pay-equity/steps/introduction/)
```

Comment:
So this link contains advice to equalize treatment--That's better than what we actually have written here, that most companies already treat people equally, which simply isn't true. Keep the link and change the text. Eg "Companies can and should work hard to ensure that **all candidates are given equal treatment** in the hiring process. But candidates should also know their worth and understand that they can negotiate to get better offers."

Reply:
(Reminder also that we can forward link here to a later discussion (TK) and links re: negotiating for equity (in treatment and compensation!) as a woman, POC, or other underrepresented employee.)